src/main/java/com/google/gwt/site/webapp/client/GWTProjectEntryPoint.java

```diff
@@ -195,118 +245,130 @@ public void f() {
     }
 
     // Use Ajax instead of default behaviour
-    $(body).on("click", "a", new Function() {
-      @Override
-      public boolean f(Event e) {
-        if (shouldEnhanceLink($(e)) &&
+    document.body.addEventListener("click", evt -> {
+      Element target = (Element) evt.target;
+        if (target.closest("a") == null) {
+          return;
+        }
+        if (shouldEnhanceLink(target) &&
             // Is it a normal click (not ctrl/cmd/shift/right/middle click) ?
-            clickHelper.handleAsClick(e)) {
+            handleAsClick((MouseEvent) evt)) {
 
           // In mobile, if menu is visible, close it
-          $("#submenu.show").removeClass("show");
+          forEach("#submenu.show", el -> el.classList.remove("show"));
 
           // Load the page using Ajax
-          loadPage($(e));
-          return false;
+          loadPage(target);
+          evt.preventDefault();
         }
-        return true;
-      }
     });
 
     // Select the TOC item when URL changes
-    $(window)
-        .on("popstate", new Function() {
-              @Override
-              public void f() {
-                loadPage(null);
-              }
-            }
-        );
+    window.addEventListener("popstate", evt -> loadPage(null));
+  }
+
+
+  public boolean handleAsClick(MouseEvent event) {
+    int mouseButtons = event.button;
+    boolean modifiers = event.altKey || event.ctrlKey || event.metaKey || event.shiftKey;
+    boolean middle = mouseButtons == 4;
+    boolean right = mouseButtons == 2;
+    return !modifiers && !middle && !right;
   }
 
-  private boolean shouldEnhanceLink(GQuery link) {
+  private boolean shouldEnhanceLink(Element link) {
     return
         // Enhance only local links
-        isSameOriginRexp.test(link.attr("href")) &&
+        isSameOriginRexp.test(link.getAttribute("href")) &&
         // Do not load links that are marked as full page reload
-        !Boolean.parseBoolean(link.attr("data-full-load"));
+        !Boolean.parseBoolean(link.getAttribute("data-full-load"));
   }
 
   private void enhanceLinks() {
     // Replace relative paths in anchors by absolute ones
     // exclude all anchors in the conHighlight and collapse menutent area.
     // TODO could be done on server side
-    $("a").not($("#content a")).each(new Function() {
-      @Override
-      public void f(Element e) {
-        GQuery link = $(e);
-        if (shouldEnhanceLink(link)) {
-          // No need to make complicated things for computing
-          // the absolute path: anchor.pathname is the way
-          link.attr("href", link.prop("pathname"));
-        }
-      }
+
+    forEach("a", el -> {
+       if (el.closest("#content") != null) {
+         return;
+       }
```

Comment:
I think you could skip the if(el.closest..) if you changed the queryselector expr to `a:not(#content a)`?